gulp/DevelopmentTasks.ts

```diff
@@ -145,7 +145,14 @@ const doc: ITaskFunction = async () => {
             commandNameSummary += " (experimental)";
         }
 
-        const anchorTag = "module-" + definition.name;
+        // create anchor tag that matches full command
+        var anchorTag = "";
```

Comment:
In js/ts, `let` and `const` are generally preferred to `var` as they have limited scope. Could you change it to `let` here instead

Reply:
Good to know. Done.